wsclient/src/test/java/org/hpccsystems/ws/client/FileSprayClientTest.java

```diff
@@ -194,7 +195,7 @@ public void testUploadFile()
         }
     }
 
-    @Test
+    @Ignore @Test
```

Comment:
I'm concerned about ignoring these tests in this manner.
Since the current assumption is that the failures are due to containerized behavior, we can disable if containerized:

We can determine if target is containerized this way:
https://github.com/hpcc-systems/hpcc4j/blob/c80a3b73e971ad3f58c8390c74de7ba2229107d0/wsclient/src/test/java/org/hpccsystems/ws/client/WSResroucesClientTest.java#L54

And invalidate the test if containerized this way:
Assume.assumeFalse("Target HPCC does not seem to be containerized", iscontainerized);